commons.webpack.config.js

```diff
@@ -95,16 +95,9 @@ var config = {
             '.tsx'
         ],
 
-        // alias: {
-        //     css: join(src, 'styles'),
-        //     containers: join(src, 'containers'),
-        //     components: join(src, 'components'),
-        //     utils: join(src, 'utils'),
-        //     styles: join(src, 'styles'),
-        //     pages: join(src, 'pages'),
-        //     shared: join(src, 'shared'),
-        //     appConfig: path.join(__dirname + '/src', 'config', ((process.env.NODE_ENV === 'test')? 'test.' : '') + 'config')
-        // }
+        alias: {
```

Comment:
how come we restored this?  just wondering.

Reply:
I import `import internalClient from 'shared/api/cbioportalInternalClientInstance';` in `persistentApiClientCache.ts`. I didn't see any harm in aliasing shared since it's shared and it makes the imports look more like they do in the main app.

Reply:
@onursumer we want to import the clients from the common library now, right?  
I also don't love the alias (even though i set it up originally myself) because I don't think IDE's can deal with them.  At least mine doesn't.  I.e. it will always try to import using relative path.

Reply:
@alisman we haven't moved cbioportal API client under the commons library because we don't use them anywhere else yet. So, they are still under `shared`. But right, ideally, we should import API clients from the `cbioportal-frontend-commons` package.